tests/vmi_multus_test.go

```diff
@@ -1240,3 +1258,35 @@ func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *
 
 	return nil
 }
+
+func defaultCloudInitNetworkData() string {
+	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should success creating default cloud init network data for SRIOV")
+	return networkData
+}
+
+func cloudInitNetworkDataWithStaticIPsByMac(macAddress, ipv4Address string) string {
+	networkData, err := libnet.NewNetworkData(
+		libnet.WithEthernet("nic0",
+			libnet.WithAddresses(ipv4Address, libnet.DefaultIPv6Address),
+			libnet.WithGateway6(libnet.DefaultIPv6Gateway),
+			libnet.WithNameserverFromCluster(),
+			libnet.WithMatchingMAC(macAddress),
+		),
+	)
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should success creating static IPs by mac address cloud init network data")
+	return networkData
+}
+
+func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipv4Address string) string {
+	networkData, err := libnet.NewNetworkData(
```

Comment:
Seems like the "default" is exactly this one, right?

Reply:
It's being used with "eth1" and not with "eth0" so it's not the default.